lua/conform/formatters/pint.lua

```diff
@@ -0,0 +1,15 @@
+local util = require("conform.util")
+
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/laravel/pint",
+    description = "Laravel Pint is an opinionated PHP code style fixer for minimalists. Pint is built on top of PHP-CS-Fixer and makes it simple to ensure that your code style stays clean and consistent.",
+  },
+  command = util.find_executable({
+    vim.fn.stdpath("data") .. "/mason/bin/pint",
```

Comment:
From what I understand of mason, it adds its bin directory to your PATH as part of its setup, so you shouldn't need this line. If `vendor/bin/pint` isn't found, falling back to `pint` should pick it up from the mason install.

Reply:
Great, thanks for the heads-up!